extensions/SemanticMediaWiki/src/MediaWiki/Jobs/JobBase.php

```diff
@@ -142,13 +141,12 @@ public static function batchInsert( $jobs ) {
 			wfDebug( __METHOD__ . " {$job->getType()}\n" . wfBacktrace() . "\n" );
 
 			$task = new \Wikia\Tasks\Tasks\JobWrapperTask();
-			// SUS-1250 - add SMW-specific tasks to a separate queue with a smaller concurency
-			$task->setQueue( SMWQueue::NAME );
 			$task->call( $job->getType(), $job->getTitle(), $job->params );
 			$tasks[] = $task;
 		}
 
-		\Wikia\Tasks\Tasks\BaseTask::batch( $tasks );
+		// SUS-1250 - add SMW-specific tasks to a separate queue with a smaller concurency
+		\Wikia\Tasks\Tasks\BaseTask::batch( $tasks, \Wikia\Tasks\Queues\SMWQueue::NAME );
```

Comment:
`\Wikia\Tasks\Tasks\BaseTask::batch` takes only one argument. Is this change fine?

Reply:
See 2090c8e - @mszabo-wikia